src/interactivity/base/InteractivityFactory.cpp

```diff
@@ -478,7 +478,18 @@ void InteractivityFactory::SetPseudoWindowCallback(std::function<void(bool)> fun
 // - <none>
 void InteractivityFactory::_WritePseudoWindowCallback(bool showOrHide)
 {
-    if (_pseudoWindowMessageCallback)
+    // BODGY
+    //
+    // GH#13158 - At least temporarily, only allow the PTY to HIDE the terminal
+    // window. There seem to be many issues with this so far, and the quickest
+    // route to mitigate them seems to be limiting the interaction here to
+    // allowing ConPTY to minimize the terminal only. This will still allow
+    // bodgy scripts to hide the Terminal via GetConsoleWindow(), but should

```

Comment:
```suggestion
    // applications to hide the Terminal via GetConsoleWindow(), but should
```

I am doing this because speel cheek didn't like it but mysteriously didn't block the build

Reply:
it only accepts capital BODGY

Reply:
I'm not merging it until the test passes so I can admin merge later